### PR TITLE
Add Dominican Republic scraper

### PR DESCRIPTION
**Pull request by romifz**
ref #340

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -25,9 +25,10 @@ pydispatcher==2.0.5       # via scrapy
 pyhamcrest==1.9.0         # via twisted
 pyopenssl==19.1.0         # via scrapy
 queuelib==1.5.0           # via scrapy
-requests==2.22.0
-scrapy==1.8.0
-scrapyd-client==1.1.0
+rarfile==3.1              # via -r requirements.in
+requests==2.22.0          # via -r requirements.in
+scrapy==1.8.0             # via -r requirements.in, scrapyd-client
```
Comment (yolile) [inline requirements.txt]: @romifz remove ", scrapyd-client" from comment

**Comment by romifz** [comment]
@jpmckinney can you please have a look at the checks that seem stuck again?

**Comment by jpmckinney** [comment]
I don't know why this happens.

Can you fix line 40 to be maximum 119 characters? I changed the flake8 test for this repo to be the same as others.

With that change Travis will pass. Is it ready to merge?

**Comment by romifz** [comment]
Done! Yes, it is ready.

**Pull request merged**